doc/Relationships.md

```diff
@@ -223,6 +223,11 @@ When displaying a post:
 
 ![embedded_list in editionView](images/embedded_list_in_editionView.png)
 
+Caution: If you want to use [`Field.template()`](./Theming.md#customizing-the-template-for-a-given-field)
```

Comment:
The docs for the "Field.template()" config in [Theming.md](https://github.com/marmelab/ng-admin/blob/faec147c4d7fcc06c0717cf4e357eeeb823e4f75/doc/Theming.md#customizing-the-template-for-a-given-field) suggest using it with an "<ma-field>" tag with 6 "magic" attrs that must be quoted verbatim, i.e.

    .template('<ma-field ng-if="entry.values.category" field="::field" value="entry.values[field.name()]" entry="entry" entity="::entity" form="formController.form" datastore="::formController.dataStore"></ma-field>', true)

Those 6 attrs expose the implementation details of ng-admin and they must be gotten exactly right for things to work. To make this even harder for users, the magic attrs that you must quote are different for "regular" fields to "embedded" fields.

Actually fixing this is out of scope here, but I thought users might benefit from a warning about this, in case they try to use "template()" on an embedded field and can't figure out why it didn't work.